README.md

```diff
@@ -37,91 +34,83 @@ If you'd like to use the accelerator-specific features of 🤗 Optimum, you can
-| Post-training Dynamic Quantization |  :heavy_check_mark:   |    :heavy_check_mark:   |          N/A            |
-| Post-training Static Quantization  |  :heavy_check_mark:   |    :heavy_check_mark:   |    :heavy_check_mark:   |
-| Quantization Aware Training (QAT)  |  Stay tuned! :star:   |    :heavy_check_mark:   |    :heavy_check_mark:   |
-| Pruning                            |        N/A            |    :heavy_check_mark:   |    :heavy_check_mark:   |
+```plain
+optimum-cli export onnx -m deepset/roberta-base-squad2 --optimize O2 roberta_base_qa
+```
 
-## Quick tour
+The model can then be quantized using `onnxruntime`:
 
-Check out the examples below to see how 🤗 Optimum can be used to train and run inference on various hardware accelerators.
+```bash
+optimum-cli onnxruntime quantize --avx512 --onnx_model roberta_base_qa
+```
 
-## Accelerated inference
+These commands will export `deepset/roberta-base-squad2` perform [O2 graph optimization](https://huggingface.co/docs/optimum/onnxruntime/usage_guides/optimization#optimization-configuration) on the exported model, and finally quantize it.
 
-#### ONNX Runtime
+For more information on the ONNX export, please check the [documentation](https://huggingface.co/docs/optimum/exporters/onnx/usage_guides/export_a_model).
 
-To accelerate inference with ONNX Runtime, 🤗 Optimum uses _configuration objects_ to define parameters for graph optimization and quantization. These objects are then used to instantiate dedicated _optimizers_ and _quantizers_.
+#### Run the exported model using ONNX Runtime
 
-Before applying quantization or optimization, first we need to load our model. To load a model and run inference with ONNX Runtime, you can just replace the canonical Transformers [`AutoModelForXxx`](https://huggingface.co/docs/transformers/model_doc/auto#transformers.AutoModel) class with the corresponding [`ORTModelForXxx`](https://huggingface.co/docs/optimum/onnxruntime/package_reference/modeling_ort#optimum.onnxruntime.ORTModel) class. If you want to load from a PyTorch checkpoint, set `from_transformers=True` to export your model to the ONNX format.
+Once the model exported to the ONNX format, we provide Python classes enabling you to run the exported file in a seemless manner using [ONNX Runtime](https://onnxruntime.ai/) in the backend:
 
 ```python
-from optimum.onnxruntime import ORTModelForSequenceClassification
 from transformers import AutoTokenizer
+from optimum.onnxruntime import ORTModelForQuestionAnswering
 
-model_checkpoint = "distilbert-base-uncased-finetuned-sst-2-english"
-save_directory = "tmp/onnx/"
-# Load a model from transformers and export it to ONNX
-tokenizer = AutoTokenizer.from_pretrained(model_checkpoint)
-ort_model = ORTModelForSequenceClassification.from_pretrained(model_checkpoint, from_transformers=True)
-# Save the ONNX model and tokenizer
-ort_model.save_pretrained(save_directory)
-tokenizer.save_pretrained(save_directory)
-```
+model_name = "roberta_base_qa"
```

Comment:
Either `roberta_base_qa_onnx/` or `roberta-base` with `export=True` IMO

Reply:
Nope, I basically take the same folder as the export command above.